packages/ui/src/Box.tsx

```diff
@@ -41,6 +41,8 @@ const ALIGN_SELF = {
   stretch: "stretch",
 };
 
+const BORDER_WIDTH = 2;
```

Comment:
Would it be a good idea to add border-width as a prop and leave this as the default?

Reply:
Oh I see, how it's set up. It's just accepting a color for the border value. This is probably fine for now.

Reply:
Yeah, the fine line I've been trying to walk here is giving just enough customization but trying to keep things pretty standard otherwise.